amd64/bsd_asm.h

```diff
@@ -40,7 +40,7 @@
 #include "../i387/osx_asm.h"
 #define CNAME(x) EXT(x)
 #else
-#include "bsd_cdefs.h"
+#include <sys/cdefs.h>
```

Comment:
That's another solution, but that would require a lot of cleanup since there are a lot of macros defined in these files (most of which I hope aren't used).